app/services/find/cycle_timetable.rb

```diff
@@ -37,6 +37,14 @@ class CycleTimetable
         find_closes: Time.zone.local(2024, 9, 30, 23, 59, 59) # The evening before Find opens in the new cycle
       },
       2025 => {
+        find_opens: Time.zone.local(2024, 10, 1, 9),
+        apply_opens: Time.zone.local(2024, 10, 8, 9),
+        first_deadline_banner: Time.zone.local(2025, 7, 12, 9),
+        apply_1_deadline: Time.zone.local(2025, 9, 7, 18),
+        apply_2_deadline: Time.zone.local(2025, 9, 21, 18),
```

Comment:
We will only have one apply deadline in 2025. I am starting to do the work to get rid of the multiple deadlines in the apply code base. I'm not sure how many methods etc will fail if you delete 1 and 2 and replace it with change this `apply_deadline`. If it is a real pain, could you maybe set the two dates to be the same (2025, 9, 21, 18)?

Reply:
Yes, this does need to be changed. 

I am planning to update them as part of this card:

https://trello.com/c/dkciMFt9/1779-update-recruitment-cycle-dates-for-rollover?filter=label:Epic%3A%20Rollover%20%2724

Reply:
Cool, if it's going to be addressed in another card I'm happy for this to go in :)